tests/e2e/archetype-plugin-e2e-tests.spec.ts

```diff
@@ -6,24 +6,25 @@ import { prepareEnvironment } from '@gmrchk/cli-testing-library';
 describe('Archetype Plugin E2E Testing for Taqueria CLI', () => {
 	test('compile offers contextual help', async () => {
 		const { execute, cleanup, exists } = await prepareEnvironment();
-		const {} = await execute('taq', 'init test-project');
+		await execute('taq', 'init test-project');
 		await exists('./test-project/.taq/config.json');
-		const {} = await execute('taq', 'install ../taqueria-plugin-archetype', './test-project');
+		await execute('taq', 'install ../taqueria-plugin-archetype', './test-project');
 		await exists('./test-project/node_modules/@taqueria/plugin-archetype/index.js');
 
 		const { stdout } = await execute('taq', 'compile --help', './test-project');
 		expect(stdout).toEqual(
 			expect.arrayContaining(['Compile a smart contract written in a Archetype syntax to Michelson code']),
 		);
+		console.log(stdout);
```

Comment:
Was this console.log for debugging purposes? Should it be removed or commented out?

Reply:
thanks, should be removed. will remove it.